### ISSUE TITLE
warning JSF1064 is logged when page refresh is forced

### ISSUE THREAD

**Issue by EugenFischer** [eclipse-ee4j/mojarra#5527]
If an impatient user frequently reloads the page, the error is logged more frequently

The guard that is supposed to prevent this is inadequate.
[ResourceHandlerImpl isConnectionAbort](https://github.com/eclipse-ee4j/mojarra/blob/master/impl/src/main/java/com/sun/faces/application/resource/ResourceHandlerImpl.java#L363).
In my case the system language is set to german so the check doesn't work.

**To Reproduce**

configure system language to any other than english
1. open any JSF page where several web resorces get loaded
2. execute F5 or Strg + F5 several times
3. warining occurs
```WARNING [jakarta.enterprise.resource.webcontainer.faces.application] (default task-14) JSF1064: Ressource theme.css aus Bibliothek primefaces kann nicht gefunden oder bedient werden.
WARNING [jakarta.enterprise.resource.webcontainer.faces.application] (default task-14) : java.io.IOException: Eine bestehende Verbindung wurde softwaregesteuert
durch den Hostcomputer abgebrochen```


**Expected behavior**
no warning get logged

![resourceHandlerImpl](https://github.com/user-attachments/assets/0219eda2-a77a-4305-aab6-dd59fd5ed8a4)

**Comment by BalusC** [comment]
Yeah, that's hacky as it hasn't been standardized yet. Ideally the underlying I/O handler should have thrown ClosedChannelException or even InterruptedIOException or EOFException instead. Try reporting an issue to Undertow to have them improve that.
